static/app/components/replays/replayTagsTableRow.tsx

```diff
@@ -0,0 +1,66 @@
+import React, {useMemo} from 'react';
+import styled from '@emotion/styled';
+import {LocationDescriptor} from 'history';
+
+import {AnnotatedText} from 'sentry/components/events/meta/annotatedText';
+import {KeyValueTableRow} from 'sentry/components/keyValueTable';
+import Link from 'sentry/components/links/link';
+import Tooltip from 'sentry/components/tooltip';
+import Version from 'sentry/components/version';
+import {EventTag} from 'sentry/types/event';
+
+type Tag = {key: string; value: string[]};
+
+interface Props {
+  tag: Tag;
+  generateUrl?: (tag: EventTag) => LocationDescriptor;
+  query?: string;
```

Comment:
we don't use these props, but w/e, they will make it easier to merge this into the original TagsTableRow component so lets leave it all alone.